### ISSUE TITLE
i want to set local path

### ISSUE THREAD

**Issue by noikiy** [banago/PHPloy#282]
i have a project but the web root is not the git root.

**Comment by plashenkov** [comment]
On git-ftp there is a --syncroot option to specify local root to be deployed.
Here we need such option too.

**Comment by banago** [comment]
@noikiy do you have to deploy to web root, now way around it? Check the copy options for a solution here: `copy[] = 'public->www'`

**Comment by plashenkov** [comment]
Hi @banago 

I've just tried the `copy[]` param and it looks like this is not a solution for this case.
Often it is necessary to deploy just one subfolder of the repository. There can be many other extra files and folders in the repository and they simply should not be deployed. There can be graphic files, non-complied sources, etc. They can be large, they can be small but countless. They can be added, removed later—and they are not to be deployed to a server. Only one subfolder is intended to be deployed.

And sometimes web root on the server is the only place where to deploy to.

This is a thing which is missed in wonderful PHPloy. I've searched the same thing in [git-deploy-php](https://github.com/BrunoDeBarros/git-deploy-php/issues/76), there is no such thing there too.
